src/Illuminate/Auth/Console/MakeAuthCommand.php

```diff
@@ -14,7 +14,7 @@ class MakeAuthCommand extends Command
      *
      * @var string
      */
-    protected $signature = 'make:auth {--views : Only scaffold the authentication views}';
+    protected $signature = 'make:auth {--views : Only scaffold the authentication views} {--tests}';
```

Comment:
Should be create another command instead? Like make:test:auth or make:auth:tests?